middleware/logger/default_logger.go

```
@@ -61,7 +61,7 @@ func defaultLoggerInstance(c fiber.Ctx, data *Data, cfg Config) error {
 			}
 
 			// Timestamp
-			buf.WriteString(data.Timestamp.Load().(string)) //nolint:forcetypeassert // Timestamp is always a string
+			buf.WriteString(data.Timestamp.Load().(string)) //nolint:forcetypeassert,errcheck // Timestamp is always a string
```

golangci-lint

directive `//nolint:forcetypeassert,errcheck // Timestamp is always a string` is unused for linter "errcheck"

(nolintlint)